### ISSUE TITLE
getting module error while using Horton package

### ISSUE THREAD

**Issue by atta1986** [theochem/horton#301]
I installed the Horton package successfully but when I tested it using "nosetests -v horton" command line it gave me the following error message:
**********
ModuleNotFoundError: No module named 'horton'
***********

I have also added it to the path in ~/.bashrc but still getting this same error. Any help would be appreciated. Thanks

**Comment by sriramsowmithri9807** [comment]
Hey @atta1986 Ensure that the Horton package is correctly installed in your Python environment and that you are using the correct Python interpreter when running the tests. You may also need to check if your environment variables are set correctly and that the `~/.bashrc` changes have been sourced properly.
Thank You.

**Comment by PaulWAyers** [comment]
@matt-chan may be able to help. Be sure that you are using the latest conda release, too.

Realistically, we suggest most people shift to HORTON 3 modules now, unless you have a specific workflow that requires HORTON 2.x.

This error, however, is probably due to environmental variables, as @sriramsowmithri9807 notes.

**Comment by sriramsowmithri9807** [comment]
@PaulWAyers yes, as you said it may the errors in environmental variables error too.

Thank you.
